www/kukd01/firma.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Tamadex</title>
+
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" media="screen" href="main.css" />
+    <script src="main.js"></script>
+</head>
+
+<body>
+    <header>
+        <!--Proč se to nezarovnává na střed ? center nesmím (validátor radí css)-->
+        <img src="IMG/Tamadex_logo.png" class="centerImage" alt="Tamadex logo">
+
+        <nav>
+            <a href="index.html">Úvodní stránka</a>
+            <a href="blog.html">Blog</a>
+            <a href="cheatsheat.html">Tahák</a>
+        </nav>
+    </header>
+
+    <!--Proč použítí spanu?-->
```

Comment:
Dobrá otázka, jsi zatim jediný kdo se na to ptal. V tvém případě to určitě nedává smysl. Já jsem měl představu, že to bude součástí headeru. Klidně to oprav na heading nebo div

Reply:
Upraveno. Díky za feedback